cypress/e2e/clue/branch/student_tests/group_test_spec.js

```diff
@@ -100,31 +105,60 @@ context('Test group functionalities', function(){
         //     });
         // });
 
-        // // TODO: Need to write tests
-        // describe('test sharing and unsharing canvases', function(){
-        //     it('verify share icon toggles correctly',()=>{
-        //         clueCanvas.getShareButton().should('have.class', 'public');
-        //         clueCanvas.shareCanvas();
-        //         clueCanvas.getShareButton().should('have.class', 'private');
-        //         clueCanvas.unshareCanvas();
-        //         clueCanvas.getShareButton().should('have.class', 'public');
-        //     });
-        //     it('will verify canvas is visible in groupmates 4-up view', function(){ //canvas is shared during set up
-        //         cy.log('need to write this test');
-        //     });
-        //     it('will unshare canvas and verify canvas is not visible in groupmates 4-up view', function(){
-        //         cy.log('need to write this test');
-        //     });
-        //     it('restore a 4-up canvas where a groupmate has shared a canvas while it was not open', function(){
-        //         cy.log('need to write this test');
-        //     });
-        //     it('restore a 4-up canvas where a groupmate has unshared a canvas while it was not open', function(){
-        //         cy.log('need to write this test');
-        //     });
-        //     it('will open a new 4-up canvas with shared canvas from other students updated', function(){
-        //         cy.log('need to write this test');
-        //     });
-        // });
+        // TODO: Need to write tests
+        describe('test sharing and unsharing canvases', function(){
+            it('verify share icon toggles correctly',()=>{
+                clueCanvas.getShareButton().should('have.class', 'public');
+                clueCanvas.shareCanvas();
+                clueCanvas.getShareButton().should('have.class', 'private');
+                clueCanvas.unshareCanvas();
+                clueCanvas.getShareButton().should('have.class', 'public');
+            });
+            // it('will verify canvas is visible in groupmates 4-up view', function(){ //canvas is shared during set up
+            //     cy.log('need to write this test');
+            // });
+            // it('will unshare canvas and verify canvas is not visible in groupmates 4-up view', function(){
+            //     cy.log('need to write this test');
+            // });
+            // it('restore a 4-up canvas where a groupmate has shared a canvas while it was not open', function(){
+            //     cy.log('need to write this test');
+            // });
+            // it('restore a 4-up canvas where a groupmate has unshared a canvas while it was not open', function(){
+            //     cy.log('need to write this test');
+            // });
+            // it('will open a new 4-up canvas with shared canvas from other students updated', function(){
+            //     cy.log('need to write this test');
+            // });
```

Comment:
Would it be too hard to add these tests in this PR?

Reply:
No @tejal-shah. Will add the test coverage as part of this PR.